undefined reference to symbol

When linking pngwriter to my code I get the following error

/usr/bin/ld: /usr/local/lib/libPNGwriter.a(pngwriter.cc.o): undefined reference to symbol 'png_set_sig_bytes@@PNG16_0'
//usr/lib/x86_64-linux-gnu/libpng16.so.16: error adding symbols: DSO missing from command line


I'm using Ubuntu 18.04 and I have libpng installed  (libpng-dev)

--- a/doc/PNGwriterQuickReference.rtf
+++ b/doc/PNGwriterQuickReference.rtf
@@ -210,7 +210,7 @@ This would plot a red dot (1 pixel) in the lower-left corner of the image, on a 
 \
 
 \f2 g++ my_program.cc -o my_program `freetype-config --cflags` \
-	-I/usr/local/include  -L/usr/local/lib -lpng -lpngwriter -lz -lfreetype
+	-I/usr/local/include  -L/usr/local/lib -lpngwriter -lpng -lz -lfreetype
 \f1 \
 \
 Otherwise, replace 

--- a/src/pngwriter.h
+++ b/src/pngwriter.h
@@ -171,7 +171,7 @@ class pngwriter
     * A typical compilation would look like this:
     *
     * g++ my_program.cc -o my_program freetype-config --cflags \
-    *          -I/usr/local/include  -L/usr/local/lib -lpng -lpngwriter -lz -lfreetype
+    *          -I/usr/local/include  -L/usr/local/lib -lpngwriter -lpng -lz -lfreetype
     *
     * If you did not compile PNGwriter with FreeType support, then remove the
     * FreeType-related flags and add -DNO_FREETYPE above.

